HARVESTER: Update Chinese translation

vickyhella opened the pull request:
- Update Chinese translation for Harvester

vickyhella commented:
@WuJun2016 Please take a look and thanks!!

WuJun2016 reviewed (commented):
@guangbochen   PTAL,  
other LGTM

guangbochen commented on pkg/harvester/l10n/zh-hans.yaml:
```diff
@@ -297,6 +300,7 @@ harvester:
       storage: 存储
       labels: 标签
       ksmtuned: Ksmtuned
+      seeder: 带外访问
```
[seeder ](https://github.com/harvester/seeder) is a project name; so you don't need to translate it.

vickyhella replied:
The English file uses `Out-of-Band access` here. Should I also change it to `Seeder` as well? @guangbochen

guangbochen commented on pkg/harvester/l10n/zh-hans.yaml:
```diff
@@ -999,15 +1022,39 @@ harvester:
+  sriov:
+    label: SR-IOV 网络设备
+    nodeName: 节点
+    numVFs: 虚拟函数数量
```
Why are we using `numVFs` here, shouldn't the UI use full words? in regards to the translation, prefer to use `虚拟数量`

WuJun2016 replied:
@DaiYuzeng   Please keep the text in the table and the dialog consistent.

DaiYuzeng replied:
https://github.com/vickyhella/dashboard/pull/2

@vickyhella I would like to send a PR into your repo: https://github.com/vickyhella/dashboard/pull/2

The column label of sr-iov VFs wil be `Number of virtual functions` now. @WuJun2016

vickyhella replied:
@DaiYuzeng's PR is merged and the translation is updated to `虚拟数量`.

vickyhella commented:
@WuJun2016 already rebased and updated the translations according to comments (and also added several new translations from the master head). Please take a second look and thanks!!

WuJun2016 reviewed (approved):
LGTM, thanks

I will modify the `seeder` key after this PR is merged.
